src/components/pages/CardView.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <card :cdata="deepUrlCard"/>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex";
+import Card from "../../components/widgets/Card";
+import router from "../../router";
+import * as actions from "../../store/actions";
+
+export default {
+  name: "creator",
+  components: { Card },
+  data() {
+    return {
+      dispatchCardCount: 0,
+      dispatchCardDeep: 0
+    };
+  },
+  computed: {
+    ...mapState(["deepUrlCard"])
+  },
+  mounted() {
+    console.log("loading Started");
+    let currentPath = this.$route.path;
+    let tokenId = currentPath.substring(
+      currentPath.lastIndexOf(":") + 1,
+      currentPath.length
+    );
+    this.$store.dispatch(actions.LOAD_DEEP_URL_CARD, { tokenId });
+    console.log("mounted");
+  },
+  methods: {
+    loadIntoStoreCard() {
```

Comment:
What triggers this method?

Reply:
nothing. removed and cleaned up the surrounding code.